k8s/base/metrics-downloader-configmap.yaml

```diff
@@ -0,0 +1,9 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: metrics-downloader
+data:
+  script.sh: |
+    DIRECTORY_NAME=$(date --date="$(date +%Y-%m-15) -1 month" +'data_%Y-%m')
```

Comment:
Date command document says this (run `info date`):

> 
> The fuzz in units can cause problems with relative items.  For
> example, '2003-07-31 -1 month' might evaluate to 2003-07-01, because
> 2003-06-31 is an invalid date.  To determine the previous month more
> reliably, you can ask for the month before the 15th of the current
> month.  For example:
> 
>      $ date -R
>      Thu, 31 Jul 2003 13:02:39 -0700
>      $ date --date='-1 month' +'Last month was %B?'
>      Last month was July?
>      $ date --date="$(date +%Y-%m-15) -1 month" +'Last month was %B!'
>      Last month was June!